motion/hybridpath_controller/hybridpath_controller/hybridpath_controller_node.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python3
+
+import rclpy
+import numpy as np
+from rclpy.node import Node
+from hybridpath_controller.adaptive_backstep import AdaptiveBackstep
+from geometry_msgs.msg import Wrench
+from nav_msgs.msg import Odometry
+from vortex_msgs.msg import HybridpathReference
+
+class HybridPathControllerNode(Node):
+    def __init__(self):
+        super().__init__("hybridpath_controller_node")
+        self.declare_parameters(
+            namespace='',
+            parameters=[
+                ('hybridpath_controller.K1_diag', [10.0, 10.0, 10.0]),
+                ('hybridpath_controller.K2_diag', [60.0, 60.0, 60.0]),
+                ('physical.inertia_matrix', [50.0, 50.0, 50.0]),
+                ('physical.damping_matrix_diag', [10.0, 10.0, 5.0])
+            ])
+        
+        self.state_subscriber_ = self.state_subscriber_ = self.create_subscription(Odometry, '/sensor/seapath/odometry/ned', self.state_callback, 1)
```

Comment:
Double check this with the seapath driver, i believe it has been changed to "/sensor/seapath/odom/ned"